dotnet/src/extensions/Azure/Handlers/BlobHandler/BlobTriggerHandler.cs

```diff
@@ -0,0 +1,203 @@
+									{
+										string propertyName = property.Name;
+										string propertyValue = property.Value.ToString();
+
+										GxUserType eventMessageProperty = CreateEventMessageProperty(eventMessPropsItemType, propertyName, propertyValue, gxcontext);
+										eventMessageProperties.Add(eventMessageProperty);
+
+									}
+								}
+
+								//Event
+
+								ClassLoader.SetPropValue(eventMessageItem, "gxTpr_Eventmessageid", messageId.ToString());
+								ClassLoader.SetPropValue(eventMessageItem, "gxTpr_Eventmessagesourcetype", EventSourceType.Blob);
+								ClassLoader.SetPropValue(eventMessageItem, "gxTpr_Eventmessagedata", myTriggerItem);
+								ClassLoader.SetPropValue(eventMessageItem, "gxTpr_Eventmessagedate", DateTime.UtcNow);
+								ClassLoader.SetPropValue(eventMessageItem, "gxTpr_Eventmessageversion", string.Empty);
+								ClassLoader.SetPropValue(eventMessageItem, "gxTpr_Eventmessageproperties", eventMessageProperties);
+
+								//List of Events
+								eventMessage.Add(eventMessageItem);
+								parametersdata = new object[] { eventMessages, null };
+							}
+							try
+							{
+								method.Invoke(objgxproc, parametersdata);
+								GxUserType EventMessageResponse = parametersdata[1] as GxUserType;//SdtEventMessageResponse
+								bool result = (bool)ClassLoader.GetPropValue(EventMessageResponse, "gxTpr_Handlefailure");
+
+								//Error handling
+
+								if (result == true) //Must retry
+								{
+									exMessage = string.Format("{0} {1}", FunctionExceptionType.AppError, ClassLoader.GetPropValue(EventMessageResponse, "gxTpr_Errormessage"));
+									throw new Exception(exMessage);
+								}
+								else
+								{
+									log.LogInformation("(GX function handler) Function finished execution.");
+								}
+							}
+							catch (Exception)
+							{
+								log.LogError("{0} Error invoking the GX procedure for Message Id {1}.", FunctionExceptionType.SysRuntimeError, messageId);
+								throw; //Throw the exception so the runtime can Retry the operation.
+							}
+						}
+					}
+					else
+					{
+						exMessage = string.Format("{0} GeneXus procedure could not be executed for Message Id {1}.", FunctionExceptionType.SysRuntimeError, messageId);
+						throw new Exception(exMessage);
+					}
+				}
+				catch (Exception)
+				{
+					log.LogError("{0} Error processing Message Id {1}.", FunctionExceptionType.SysRuntimeError, messageId);
+					throw; //Throw the exception so the runtime can Retry the operation.
+				}
+			}
+			else
+			{
+				exMessage = string.Format("{0} GeneXus procedure could not be executed while processing Message Id {1}. Reason: procedure not specified in configuration file.", FunctionExceptionType.SysRuntimeError, messageId);
+				throw new Exception(exMessage);
+			}
+		}
+		private GxUserType CreateEventMessageProperty(Type eventMessPropsItemType, string propertyId, object propertyValue, GxContext gxContext)
```

Comment:
This routine is repeated in each handler. If possible, all handlers could extend a base common class that contains this routine.

Reply:
Fixed for this routine. I will do the changes for the other handlers in another PR.